eZ/Publish/Core/Persistence/Legacy/Tests/Content/ContentHandlerTest.php

```diff
@@ -108,6 +109,11 @@ class ContentHandlerTest extends TestCase
      */
     protected $contentTypeHandlerMock;
 
+    /**
+     * @var \eZ\Publish\Core\Persistence\Legacy\Content\Language\Handler
+     */
+    protected $languageHandlerMock;
```

Comment:
Prefer `private`, unless we have a reason to give the extending class access to this service (descending class can declare it's own private field anyway, if needed).
```suggestion
    private $languageHandlerMock;
```